scipy/stats/_distn_infrastructure.py

```diff
@@ -1702,6 +1702,11 @@ def _get_fixed_fit_value(kwds, names):
                          ', '.join(repeated))
     return vals[0][1] if vals else None
 
+
+class FitError(RuntimeError):
```

Comment:
If we have custom errors, they should be available in the public API so that user can rely on it in their `try, except` blocks.

Reply:
This is a good idea, but yes, part of a broader issue. See gh-13462. As noted there, adding `FitError` to the public API would require approval from the mailing list. Actually, gh-13462 would suggest that instead of `FitError` itself being public, it should probably be a subclass of a more general public error like `SciPyStatsError` to minimize the number of separate errors in the API.

Shall we merge this and I'll tackle that issue next?

Reply:
Just seeing this issue. Yes it's a good idea to just expose a few errors and do some subclassing. Only drawback of the subclassing is that we need to document what is the base class so users can catch the correct one. Ok to merge now, as you will continue working on this. In the end we just need to ensure we don't advertise for `FitError` if not part of the API.